src/herder/HerderImpl.cpp

```diff
@@ -125,14 +125,25 @@ HerderImpl::getStateHuman() const
 void
 HerderImpl::bootstrap()
 {
-    CLOG(INFO, "Herder") << "Force joining SCP with local state";
-    assert(getSCP().isValidator());
-    assert(mApp.getConfig().FORCE_SCP);
+    if (!mBootstrapped)
+    {
+        mBootstrapped = true;
+
+        CLOG(INFO, "Herder") << "Force joining SCP with local state";
+        assert(getSCP().isValidator());
+
+        if (!mApp.getConfig().FORCE_SCP)
+        {
+            auto const& lcl = mLedgerManager.getLastClosedLedgerHeader();
+            mHerderSCPDriver.restoreSCPState(lcl.header.ledgerSeq,
+                                             lcl.header.scpValue);
+        }
 
-    mLedgerManager.bootstrap();
-    mHerderSCPDriver.bootstrap();
+        mLedgerManager.bootstrap();
```

Comment:
I am really not sure this is right.
we're introducing a race between whatever that node is doing and calling `bootstrap`: those bootstrap calls may reset a bunch of things that the node was actually doing.

I thought the less dangerous version of this PR was to just allow `manualclose` to be called at any time?